src/Views/Components/BoulderCard.tsx

```diff
@@ -23,19 +23,17 @@ export const BoulderCard = ({ boulderE }: BoulderCardProps) => {
   const gNr = config.databaseUrl.includes("quadrel") ? config.grades.indexOf(grade) + 1 : gradeNr;
 
   return (
-    <Link href={`/boulder/${boulderE.objectId}`} legacyBehavior>
-      <Card>
-        <BoulderId $grade={gNr >= 1000 ? "pink" : gradeColor}>{gNr >= 1000 ? "?" : gNr}</BoulderId>
-        <Meta>
-          <Sector>{config.prettyPrintSector(sector)}</Sector>
-          <Setters>
-            {setter.map((setterId, index) => (
-              <BoulderCardSetter key={index} setterId={setterId} />
-            ))}
-          </Setters>
-        </Meta>
-      </Card>
-    </Link>
+    <Card as={Link} href={`/boulder/${boulderE.objectId}`}>
```

Comment:
This Card components needs text-decoration:none (and other anchor style resets, if needed). Otherwise it'll look like this: 

<img width="200" height="95" alt="Image" src="https://github.com/user-attachments/assets/b5782bcb-622e-4a06-bbda-6edf08533369" />